tests/Model/ModelManagerTest.php

```diff
@@ -146,6 +146,13 @@ public function testInstantiateWithDeprecatedRegistryInterface(): void
         $this->assertSame($em, $this->modelManager->getEntityManager('x'));
     }
 
+    public function testSupportsQuery(): void
+    {
+        $this->assertTrue($this->modelManager->supportsQuery($this->createMock(ProxyQuery::class)));
```

Comment:
Can you use a data provider for this? This test covers multiple cases.

Reply:
Sure, but I had difficult to see the interest in a dataprovider here since it leads with more complexity/code for such an easy test.

Reply:
Done @core23